src/one_dragon_qt/widgets/setting_card/value_display_editable_combo_box_setting_card.py

```
@@ -0,0 +1,190 @@
+)
+
+
+class ValueDisplayEditableComboBoxSettingCard(EditableComboBoxSettingCard):
+    """可输入真实值，并在右侧显示匹配选项外显名的下拉设置卡片。"""
+
+    def __init__(
+            self,
+            icon: str | QIcon | FluentIconBase,
+            title: str,
+            content: str = '',
+            icon_size: IconSize = IconSize(16, 16),
```

Ruff (0.15.12)

[warning] Do not perform function call `IconSize` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)